Dockerfile

```
@@ -37,18 +37,20 @@ RUN ./gradlew clean -x rat -x test war
 
 # =========================================
 
-FROM bitnami/tomcat:7.0.94 as fineract
+FROM bitnami/tomcat:latest as fineract
```

```suggestion
FROM bitnami/tomcat:9.0 as fineract
```

because `:latest` is always dangerous (it could silently break us when Tomcat 10 is released)

---

@ptuomola did you mean to Commit Suggestion to accept this, or manually work this into your Commit? This Conversation was resolved, and as I was looking this over about to merge it, I've noticed that it hasn't actually been changed... or perhaps I'll just merge this, and raise a quick follow-up PR myself proposing to change it.

---

Sorry. I did Commit Suggestion to accept it, but then looks like when I pushed a new commit with the other changes you suggested (i.e. README.md etc) that got overwritten. I've now pushed a new commit that hopefully addresses all of the comments you raised. Thanks again for all the feedback!